src/status_im/ui/screens/profile/user/styles.cljs

```diff
@@ -1,6 +1,80 @@
-(ns status-im.ui.screens.profile.user.styles)
+(ns status-im.ui.screens.profile.user.styles
+  (:require [quo2.foundations.colors :as colors]
+            [react-native.safe-area :as safe-area]))
 
 (def share-link-button
   {:margin-top        12
    :margin-horizontal 16
    :margin-bottom     16})
+
+(def radius 16)
+
+(def top-background-view
+  {:background-color colors/magenta-opa-40
+   :position         :absolute
+   :top              0
+   :left             0
+   :height           400
+   :width            400
```

Comment:
any particular reason for 400? would`:bottom 0 :right 0` as well not be better? 🤔

Reply:
Just want to keep the view as the solid background (magenta) on the top when pull down. 
if use bottom 0, the magenta background is visible when scroll to the bottom.